deployer/src/deployment/deploy_layer.rs

```diff
@@ -30,7 +31,7 @@ use super::{
 };
 
 /// Records logs for the deployment progress
-pub trait LogRecorder {
+pub trait LogRecorder: Clone + Send + 'static {
```

Comment:
Not that it matters too much, but `LogRecorder` seems very close to [`futures::sink::Sink::<Log>`](https://docs.rs/futures/latest/futures/sink/trait.Sink.html).

Reply:
Do you see a benefit for us converting this to a `Sink` (like buffering)? Maybe I should add it as a ticket under the deployer project?